modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/exportimport/portlet/preferences/processor/DDMFormExportImportPortletPreferencesProcessor.java

```diff
@@ -126,12 +126,16 @@ public PortletPreferences processExportPortletPreferences(
 			return portletPreferences;
 		}
 
+		if (ExportImportThreadLocal.isStagingInProcess() &&
```

Comment:
@joseabelenda, how about reuse the previous if to attend your needs? For example:
```
if (ExportImportThreadLocal.isStagingInProcess()) {
    if (!group.isStagedPortlet(DDMPortletKeys.DYNAMIC_DATA_MAPPING_FORM_ADMIN)) {
        ...
    }

    if (!(group.isStagingGroup() || group.isCompanyStagingGroup())) {
        ...
    }
}
```